front/components/assistant/conversation/ConversationMessage.tsx

```diff
@@ -51,12 +51,17 @@ export function EmojiSelector({
           />
         </div>
       </DropdownMenu.Button>
-      <DropdownMenu.Items width={400} origin="topRight" overflow="visible">
-        <Picker
+      <DropdownMenu.Items
```

Comment:
Fixing the display:

**Before:**

<img width="446" alt="image" src="https://github.com/dust-tt/dust/assets/7428970/fa7e0975-2935-4be8-ab4b-5fd09a34b6ed">

**After:**

<img width="378" alt="image" src="https://github.com/dust-tt/dust/assets/7428970/ac7cc4db-00b3-43a2-b0de-680fb71bfcd2">